st2actions/st2actions/query/mistral/v2.py

```diff
@@ -42,13 +42,25 @@ def query(self, execution_id, query_context):
             status, output = self._get_workflow_result(exec_id)
             if output and 'tasks' in output:
                 LOG.warn('Key conflict with tasks in the workflow output.')
+        except requests.exceptions.ConnectionError:
+            msg = 'Unable to connect to mistral.'
+            LOG.exception(msg)
+            return (LIVEACTION_STATUS_RUNNING, {'error': msg})
```

Comment:
This is not a good solution at all. This will just let us know if there was a mistral connection error. If the connection reestablishes in a subsequent run, we would get the results back.